src/contracts/examples/fastlane-online/OuterHelpers.sol

```diff
@@ -259,21 +276,23 @@ contract OuterHelpers is FastLaneOnlineInner {
         return sortedSolverOps;
     }
 
-    function _updateSolverReputation(
-        SolverOperation[] memory solverOps,
-        uint128 magnitude,
-        bool solversSuccessful
-    )
-        internal
-    {
+    function _updateSolverReputation(SolverOperation[] memory solverOps, uint128 magnitude) internal {
         uint256 _length = solverOps.length;
+        address _solverFrom;
+
         for (uint256 i; i < _length; i++) {
-            if (solversSuccessful) {
-                S_solverReputations[solverOps[i].from].successCost += magnitude;
+            _solverFrom = solverOps[i].from;
+
+            // winningSolver will be address(0) unless a winning solver fulfilled the swap intent.
+            if (_solverFrom == _getWinningSolver()) {
+                S_solverReputations[_solverFrom].successCost += magnitude;
```

Comment:
We should add a `break;` here (after the successCost increment) so that solvers who bid lower than the winning solver don't have their submission counted as a failure.

Reply:
Good catch, fixed here https://github.com/FastLane-Labs/atlas/pull/400/commits/1b97be751e321df43c9db0768e001dfdeb8199af